home/src/main/resources/rdf/i18n/en_US/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -254,6 +254,102 @@ uil-data:reset_password_note.Vitro
         uil:hasKey      "reset_password_note" ;
         uil:hasPackage  "Vitro-languages" .
 
+uil-data:password_reset_email_sent.Vitro
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "Password recovery email is sent to "@en-US ;
+        uil:hasApp      "Vitro" ;
+        uil:hasKey      "password_reset_email_sent" ;
+        uil:hasPackage  "Vitro-languages" .
+
+uil-data:password_reset_email_sent_if_exists.Vitro
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       ", if it is bound with an existing account."@en-US ;
```

Comment:
", if it is _associated with_ an existing account."

Reply:
Done